pom.xml

```diff
@@ -696,6 +697,11 @@
         <artifactId>junit</artifactId>
         <version>${junit.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.tngtech.java</groupId>
```

Comment:
usually it is better to not add such third party dependencies (tngtech ??) if not strictly needed

in JUnit4 this is the way to do it
https://github.com/junit-team/junit4/wiki/Parameterized-tests